tests/e2e/core-tests/specs/merchant/wp-admin-order-status-filter.test.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+const {
+	StoreOwnerFlow,
+	createSimpleOrder,
+	clickFilter,
+	moveAllItemsToTrash,
+} = require( '@woocommerce/e2e-utils' );
+
+let statusColumnTextSelector = 'mark.order-status > span';
+
+const runOrderFiltersTest = () => {
+	describe('WooCommerce Orders > Filter Orders by Status', () => {
+		beforeAll(async () => {
+			// First, let's login
+			await StoreOwnerFlow.login();
+
+			// Next, let's create some orders we can filter against
+			await createSimpleOrder('Pending payment');
```

Comment:
Noting here that we can add the constant to `tests/e2e/env/config/default.json` as order statuses can be used across other tests as well so it will be useful to have it in the default file.

Reply:
I like that idea--I ended up deciding to add these to the `default.json` file, like so:

```json
  "orderstatuses": {
    "pendingpayment": "Pending payment",
    "processing": "Processing",
    "onhold": "On hold",
    "completed": "Completed",
    "cancelled": "Cancelled",
    "refunded": "Refunded",
    "failed": "Failed"
  }
```

Reply:
@zhongruige @juliaamosova There is a three part issue with putting the statuses in the config file

- There are 4 packages `tests/e2e/(api|core-tests|env|utils)`. Constants for the core tests should either be in `core-tests` or `utils` depending on whether they are only useful in core tests or could be used in testing in other projects. `e2e-core-tests` does not have a dependency on `e2e-environment` and can be installed in a project without it.

- `tests/e2e/env/config/default.json` is the default configuration file used when `tests/e2e/config/default.json` does not exist.
- `default.json` describes the test environment and provides a method for test environments to alter test parameters. We shouldn't add any data to it that WC core wants to retain control of. If at some point we wanted to implement support for translations we would likely choose an alternative to adding all the strings to this configuration file.

Reply:
I can see that--we want to leave the config files more for configurable aspects of the tests; whereas (in this example) strings that are reliant on the system under test we'll want defined separately.

So, going back to your [original example](https://github.com/woocommerce/woocommerce/pull/28437#discussion_r535296006) @rrennick, would you prefer these `const` values for order statuses defined within the test, or somewhere else? I may have overlooked it, but I didn't see a specific file within the tests that defined constants from the UI. So I am not sure if that is something we want to add on this task, or if we'll want to add it as we start to have the need to add more of these types of values in the future. Let me know what you think.

Reply:
> would you prefer these const values for order statuses defined within the test, or somewhere else?

In that file since that is the first use. We will probably want to discuss whether they should live in the core tests or utils. We don't want to hold up this PR pending that discussion.

Reply:
Sounds good! I've pushed up those changes :+1:

Reply:
@zhongruige Nice work. I've resolved both my requests on the PR.

Reply:
Thanks for the feedback @rrennick!